app/middlewares/wsAuth.js

```diff
@@ -0,0 +1,17 @@
+let jwt = require('jsonwebtoken');
+
+let config = require('../config/index');
+let errors = require('../config/errors');
+
+let UserDto = require('../dtos/UserDto');
+
+module.exports = function wsAuthMiddleware(socket, next) {
+  let token = socket.handshake.query.token;
```

Comment:
I think we should check if there's no token at all and return noToken error, similar to auth middleware.